xml/admin_rbd.xml

```diff
@@ -336,9 +381,7 @@
        In the following example you can see how to specify a user name and a
        keyring with a corresponding secret:
       </para>
-<screen>
-&prompt.cephuser;rbdmap map mypool/myimage id=rbd_user,keyring=/etc/ceph/ceph.client.rbd.keyring
-</screen>
+<screen>&prompt.cephuser;rbdmap map mypool/myimage id=<replaceable>rbd_user</replaceable>,keyring=/etc/ceph/ceph.client.rbd.keyring</screen>
```

Comment:
probably longer than 80 chars?

Reply:
Do you mean that you think it should have a linebreak inserted too? It displays without horizontal scrolling at a comparable width to the text around it, for me.